spec/templater_spec.rb

```diff
@@ -24,29 +24,31 @@
 
 require 'templater'
 require 'spec_utils'
-require 'fakefs_helper'
 require 'node'
+require 'filesystem'
 
-TEST_TEMPLATE_PATH = File.join(FIXTURES_PATH, 'template.erb')
-REPO_TEST_CONFIG_PATH = File.join(FIXTURES_PATH, 'configs/repo-unit-test.yaml')
-UNSET_PARAMETER_TEMPLATE_PATH = File.join(FIXTURES_PATH, 'unset_parameter_template.erb')
+TEST_TEMPLATE_PATH = '/fixtures/template.erb'
+UNSET_PARAMETER_TEMPLATE_PATH = '/fixtures/unset_parameter_template.erb'
 TEST_HUNTER_PATH = File.join(FIXTURES_PATH, 'cache/hunter.yaml')
 
 
 RSpec.describe Metalware::Templater do
-  def expect_renders(template_parameters, expected, config: Metalware::Config.new)
-    fshelper = FakeFSHelper.new(config)
-    fshelper.clone(FIXTURES_PATH)
-    fshelper.clone_repo(config.repo_path)
+  let :filesystem {
+    FileSystem.setup do |fs|
+      fs.with_fixtures('/', at: '/fixtures')
```

Comment:
`FileSystem#with_fixture` is a wrapper around `FakeFS::FileSystem.clone`, which can run whether or not the `FakeFS` is activated and will clone files into it; I believe it bypasses `FakeFS`'s own replacements of various methods for interacting with the file system. So this works fine whether you run it in `setup`, `test`, or on a `FileSystemConfigurator directly. You can see this if you comment this line, then the tests will fail with various errors like this:

```
     Errno::ENOENT:
       No such file or directory - /fixtures/template.erb
```